lib/graphql/utils.ex

```diff
@@ -65,8 +73,18 @@ defmodule BridgeEx.Graphql.Utils do
   def parse_response({:error, error}), do: {:error, error}
 
   def parse_response({:ok, %{errors: errors}}), do: {:error, errors}
+  def parse_response({:ok, %{"errors" => errors}}), do: {:error, errors}
 
   def parse_response({:ok, %{data: data}}), do: {:ok, data}
+  def parse_response({:ok, %{"data" => data}}), do: {:ok, data}
+
+  def atom_decoder(body), do: Jason.decode(body, keys: :atoms)
```

Comment:
Could these be private?

Reply:
The decoders in this file could be used from outside, if one prefers to be explicit on what the code does use. I added the atoms after using function passing for a bit, just as a "simpler" way of defining the module when you are simply parsing JSONs. If you believe that using atoms is more readable, we can totally make those functions private.

Reply:
I think it would be simpler especially for the future, if we let the user access these decoders then we would have to consider them as part of our API and if we want to change something we'll have to issue breaking changes and so on

Reply:
Yeah you are right, going to change them now :smile: thanks!